src/lib.rs

```diff
@@ -784,6 +799,13 @@ impl<'a> ParserState<'a> {
                 b'M' => Operator::EHVectorDtorIterator,
                 b'N' => Operator::EHVectorVBaseCtorIterator,
                 b'O' => Operator::CopyCtorClosure,
+                b'R' => {
+                    let c = self.get()?;
+                    match c {
+                        b'0' => Operator::RTTITypeDescriptor,
```

Comment:
I'd rather we do the data type parsing here and include the data type as data in the Operator enum. That will let us avoid the special case in read_name() and lets us parse things like '??_R0E@8' the same way as undname. Just leave the '8' to be handled as the symbol type.

Reply:
So you want something like:

```rust
match c {
  // Multiple RTTI symbols get here, so if we want something more specific,
  // we have to check the symbol name.
  '8' => Type::RTTIThing
}
```

when parsing, and then when printing, printing the operator already has the type at hand (contained in the `Operator` enum) so it can print that directly, and printing the `Type::RTTIThing` is essentially a no-op?

Reply:
Yep